### PR TITLE
Add support for TIMESYS

### PR DESCRIPTION
**Pull request by judith-ipac**
Resource_Element: Moved <table_elements_> to <options_>.
Table: Removed assorted complications rendered unnecessary by recent changes in query_server.
Table, Resource_Element: Exposed attribute setters.
read_resource_element(): Now check for 'TIMESYS' tag when parsing elements.
read/write_fits():  Improved handling of attributes.

Updated tests.

### CODE REVIEW

File: src/Table/Table.cxx [hunk 1]
```diff
@@ -311,62 +314,44 @@ bool Table::add_attributes_labeled_by_element(
...
-        boost::equals(label, INFO)) {
+    if (is_property_style_label(label)) {
         get_labeled_properties().emplace_back(label_and_prop);
         return;
     }
     if (get_resource_elements().empty()) {
         throw std::runtime_error(
                 "This function cannot be used to add resource-level properties to "
-                "empty table.");
+                "embryonic table without resource_elements.");
```
Comment (smonkewitz) [inline src/Table/Table.cxx:328]: "to an embryonic table" or just "to a table"

**Review by smonkewitz — APPROVED** [review]
I have a few minor comments, but looks good to me!

File: src/Resource_Element.hxx [hunk 2]
```diff
@@ -13,6 +13,10 @@
 namespace tablator {
 
 static constexpr size_t MAIN_TABLE_ELEMENT_IDX = 0;
+static const std::string RESULTS("results");
+static const std::string META("meta");
+static const std::string OTHER("other");
+static const std::string NONE("none");
```
[the remark this answers is not in the document]
Comment (judith-ipac) [reply src/Resource_Element.hxx:19]: I think I am finally internalizing this syntax.

File: src/Resource_Element.hxx [hunk 3]
```diff
@@ -23,29 +27,46 @@ enum class Resource_Type {
...
+        case Resource_Type::RESULTS:
+            return RESULTS;
+        case Resource_Type::META:
+            return META;
+        case Resource_Type::NONE:
+            return NONE;
+        default:
+            return OTHER;
+    }
+    return OTHER;
```
[the remark this answers is not in the document]
Comment (judith-ipac) [reply src/Resource_Element.hxx:62]: Yes, placate.

**Pull request merged**